### DISCUSSION TITLE
How to record exception without Stack Trace

### DISCUSSION THREAD

**Discussion by Hieu-Nguyen-1** [open-telemetry/opentelemetry-dotnet#3996] [Ideas]
Dear all,

How can I record exception without stacktrace

I am using:

.AddHttpClientInstrumentation(options => options.RecordException = true)
.AddAspNetCoreInstrumentation(options => options.RecordException = true)
.AddSqlClientInstrumentation(options => options.RecordException = true);

Thanks

**Comment by cijothomas** [comment]
There is no option to opt-out of stacktrace today.
Are you concerned about the size? or something else? It might be something we need to add a opt-out switch or opti-in switch.
